File: HealthSysHub.Web.Utility/Models/ResetPassword.cs
namespace HealthSysHub.Web.Utility.Models
{
    public class ResetPassword
    {
        public Guid? Id { get; set; }
        public Guid? StaffId { get; set; }
        public string Password { get; set; }
        public Guid? ModifiedBy { get; set; } 
        public DateTimeOffset? ModifiedOn { get; set; } 
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/resetpassword.ts
export interface ResetPassword {
  id?: string;          // Use string for GUID representation in TypeScript
  staffId?: string;     // Use string for GUID representation in TypeScript
  password: string;
  modifiedBy?: string;  // Use string for GUID representation in TypeScript
  modifiedOn?: Date;    // Use Date for DateTimeOffset representation in TypeScript
}